H2TauTau/python/proto/analyzers/TreeVariables.py

```diff
@@ -150,15 +151,27 @@ def __init__(self, name, function=None, type=float):
     # JAN - only one PU mva working point, but we may want to add more
     # run in our skimming step
     # (for which Jet.py would have to be touched again)
-    Variable('mva_pu', lambda jet : jet.puMva('pileupJetId:fullDiscriminant')),
-    Variable('id_loose', lambda jet : jet.looseJetId()),
-    Variable('id_pu', lambda jet : jet.puJetId()),
-    Variable('mva_btag', lambda jet : jet.btagMVA),
-    Variable('area', lambda jet : jet.jetArea()),
-    Variable('flavour_parton', lambda jet : jet.partonFlavour()),
-    Variable('csv', lambda jet : jet.btagMVA),
-    Variable('rawfactor', lambda jet : jet.rawFactor()),
-    Variable('genjet_pt', lambda jet : jet.matchedGenJet.pt() if hasattr(jet, 'matchedGenJet') and jet.matchedGenJet else -999.),
+    Variable('mva_pu'                   , lambda jet : jet.puMva('pileupJetId:fullDiscriminant') if hasattr(jet, 'puMva'                    ) else -999.                      ),
```

Comment:
I know it's me not being in line with the general tastes, but I find pep8 often hard to read.
But I'll try to stick to it, ok.